src/assets/CollectiblesController.test.ts

```diff
@@ -181,387 +187,491 @@ describe('CollectiblesController', () => {
-      name: 'KudosToken',
-      symbol: 'KDO',
+      await collectiblesController.addCollectible(ERC721_KUDOSADDRESS, '1203');
+      expect(collectiblesController.state.collectibles[0]).toStrictEqual({
+        address: ERC721_KUDOSADDRESS,
+        image: 'Kudos Image (from uri)',
+        name: 'Kudos Name (from uri)',
+        description: 'Kudos Description (from uri)',
+        tokenId: '1203',
+        collectionImage: 'collection.url',
+        collectionName: 'Collection Name',
+        imageOriginal: 'Kudos url',
+        standard: 'ERC721',
+      });
+
+      expect(
+        collectiblesController.state.collectibleContracts[0],
+      ).toStrictEqual({
+        address: ERC721_KUDOSADDRESS,
+        name: 'KudosToken',
+        symbol: 'KDO',
+      });
     });
-  });
 
-  it('should add collectible erc721 and get collectible contract information only from contract', async () => {
-    assetsContract.configure({ provider: MAINNET_PROVIDER });
+    it('should add collectible erc721 and get collectible contract information only from contract', async () => {
+      assetsContract.configure({ provider: MAINNET_PROVIDER });
 
-    sandbox
-      .stub(
-        collectiblesController,
-        'getCollectibleContractInformationFromApi' as any,
-      )
-      .returns(undefined);
+      sandbox
+        .stub(
+          collectiblesController,
+          'getCollectibleContractInformationFromApi' as any,
+        )
+        .returns(undefined);
 
-    sandbox
-      .stub(collectiblesController, 'getCollectibleInformationFromApi' as any)
-      .returns(undefined);
-    await collectiblesController.addCollectible(ERC721_KUDOSADDRESS, '1203');
-    expect(collectiblesController.state.collectibles[0]).toStrictEqual({
-      address: ERC721_KUDOSADDRESS,
-      image: 'Kudos Image (from uri)',
-      name: 'Kudos Name (from uri)',
-      description: 'Kudos Description (from uri)',
-      tokenId: '1203',
-      standard: 'ERC721',
-    });
+      sandbox
+        .stub(collectiblesController, 'getCollectibleInformationFromApi' as any)
+        .returns(undefined);
+      await collectiblesController.addCollectible(ERC721_KUDOSADDRESS, '1203');
+      expect(collectiblesController.state.collectibles[0]).toStrictEqual({
+        address: ERC721_KUDOSADDRESS,
+        image: 'Kudos Image (from uri)',
+        name: 'Kudos Name (from uri)',
+        description: 'Kudos Description (from uri)',
+        tokenId: '1203',
+        standard: 'ERC721',
+      });
 
-    expect(collectiblesController.state.collectibleContracts[0]).toStrictEqual({
-      address: ERC721_KUDOSADDRESS,
-      name: 'KudosToken',
-      symbol: 'KDO',
+      expect(
+        collectiblesController.state.collectibleContracts[0],
+      ).toStrictEqual({
+        address: ERC721_KUDOSADDRESS,
+        name: 'KudosToken',
+        symbol: 'KDO',
+      });
     });
-  });
 
-  it('should add collectible by selected address', async () => {
```

Comment:
It looks like this test is no longer preset. Was it removed accidentally?

I see that there is now an equivalent test for your new method, but not for `addCollectible`, which is what this tested

Reply:
Yes I replaced it with a test for my method. I will add it back for `addCollectible` and rename it slightly and keep it for testing the new method as well? Sound good @Gudahtt?